### PR TITLE
fix: [WIN-NPM] back-compat mitigation to PATH issue

### PR DESCRIPTION
**Pull request by huntergregory**
**Reason for Change**:
Resolve PATH issue correctly. Removing `powershell.exe` is **back-compat** with NPM v1.3.34 (in-prod).

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:
Don't want to override PATH variable.

#### Error
```
powershell.exe : The term 'powershell.exe' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
At line:1 char:27
+ .\setkubeconfigpath.ps1 ; powershell.exe .\npm.exe start --kubeconfig ...
+                           ~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (powershell.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

### CODE REVIEW

**Review by vakalapa — CHANGES_REQUESTED** [review]
@rbtr @matmerr  wdyt ?

File: npm/examples/windows/azure-npm.yaml
Comment (huntergregory) [inline npm/examples/windows/azure-npm.yaml:1]: appending with %PATH% doesn't actually append anything

File: npm/examples/windows/azure-npm.yaml [hunk 1]
```diff
@@ -109,7 +109,7 @@ spec:
             - name: NPM_CONFIG
               value: .\\etc\\azure-npm\\azure-npm.json
             - name: PATH
-              value: '%CONTAINER_SANDBOX_MOUNT_POINT%\Windows\System32\WindowsPowershell\v1.0\'
+              value: '%CONTAINER_SANDBOX_MOUNT_POINT%\Windows\System32\WindowsPowershell\v1.0\;C:\Windows\System32\'
```
[the remark this answers is not in the document]
Comment (huntergregory) [reply npm/examples/windows/azure-npm.yaml]: appending with %PATH% doesn't actually append anything

File: npm/examples/windows/azure-npm.yaml [hunk 2]
```diff
@@ -108,8 +107,6 @@ spec:
                   fieldPath: spec.nodeName
             - name: NPM_CONFIG
               value: .\\etc\\azure-npm\\azure-npm.json
-            - name: PATH
```
Comment (huntergregory) [inline npm/examples/windows/azure-npm.yaml:111]: remove our first attempt at mitigation, which is not back-compat and is incomplete (can't use vfpctrl and other tools used by HNS to collect logs)

**Review by vakalapa — APPROVED** [review]

**Pull request merged**